public/index.html

```diff
@@ -17,59 +18,62 @@ <h1 class="header__wordmark">
           fluent
         </h1>
         <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          <a class="link" href="http://projectfluent.io/fluent/guide">Learn</a>
+          <a class="link" href="http://projectfluent.io/play">Play</a>
+          <a class="link" href="#implementation-docs">Use</a>
+          <a class="link" href="https://github.com/projectfluent">Contribute</a>
+          <!-- <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Community</a> -->
         </nav>
       </header>
 
       <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
-        </p>
+        <div class="pitch-wrapper">
+          <p class="pitch">
+            <strong>Fluent</strong> is a set of localization libraries<br>
+            designed to create better translations.
```

Comment:
I'd like to avoid potential confusion here: Fluent doesn't _create_ translations.  It help author them.  How about:

    Fluent is a family of localization libraries
    designed to unleash the expressive power
    of the natural language.

Or:

    Fluent is a family of localization libraries
    designed to improve how software is translated.

Or:

    Fluent is a family of localization libraries
    designed to help create better translations.

Reply:
I like all of these. Second one is my favourite.